scripts/run_frontend_tests.sh

```diff
@@ -51,6 +51,8 @@ export DEFAULT_SKIP_INSTALLING_THIRD_PARTY_LIBS=false
 export DEFAULT_RUN_MINIFIED_TESTS=false
 maybeInstallDependencies "$@"
 
+find . -type f \( -name "*.spec.ts" ! -name "state-interaction-editor.directive.spec.ts" -o -name "*Spec.ts" \) -exec cat {} \; > ./core/templates/dev/head/combined-tests.spec.ts
```

Comment:
Add a clear, fully-detailed comment to explain what is going on here and why we need to do this.

Reply:
Done!